src/components/header/headerNavbar.tsx

```diff
@@ -63,78 +65,108 @@ const HeaderNavbar = () => {
+                        end={true}
+                        className={({ isActive }) =>
+                          isActive ? activePageStyle : inactivePageStyle
+                        }
+                      >
+                        Home
+                      </NavLink>
+                    </Col>
+                    <Col xs={navColsSpanXS} className={navColsClasses}
+                    onClick={() => handleCloseOffCanvas()}>
+                      <NavLink
+                        to="/browse"
+                        className={({ isActive }) =>
+                          isActive ? activePageStyle : inactivePageStyle
+                        }
+                      >
+                        Browse Data
+                      </NavLink>
+                    </Col>
+                    <Col xs={navColsSpanXS} className={navColsClasses}
+                    onClick={() => handleCloseOffCanvas()}>
+                      <NavLink
+                        to="/download"
+                        className={({ isActive }) =>
+                          isActive ? activePageStyle : inactivePageStyle
+                        }
+                      >
+                        Access Data
+                      </NavLink>
+                    </Col>
+                    <Col xs={navColsSpanXS} className={navColsClasses}
+                    onClick={() => handleCloseOffCanvas()}>
+                      <NavLink
+                        to="/upload"
+                        className={({ isActive }) =>
+                          isActive ? activePageStyle : inactivePageStyle
+                        }
+                      >
+                        Submit Data
+                      </NavLink>
+                    </Col>
+                    <Col xs={navColsSpanXS} className={navColsClasses}
+                    onClick={() => handleCloseOffCanvas()}>
+                      <NavLink
+                        to="/metadata-model"
+                        className={({ isActive }) =>
+                          isActive ? activePageStyle : inactivePageStyle
+                        }
+                      >
+                        Metadata Model
+                      </NavLink>
+                    </Col>
+                    <Col xs={navColsSpanXS} className={navColsClasses}
+                    onClick={() => handleCloseOffCanvas()}>
+                      <NavLink
+                        to="/faq"
+                        className={({ isActive }) =>
+                          isActive ? activePageStyle : inactivePageStyle
+                        }
+                      >
+                        FAQ
+                      </NavLink>
+                    </Col>
+                  </Nav>
+                </Col>
+                <Col
+                  className="justify-content-center d-flex px-0 pe-xl-2 pe-xxl-5"
+                  xs={12}
+                  lg={3}
+                  xl={3}
+                >
```

Comment:
```suggestion
                <Col
                  className={
                    "justify-content-center d-flex px-0" +
                    (showOffCanvas ? " pt-sm-3" : " pe-xl-2 pe-xxl-5")
                  }
                  xs={12}
                  lg={showOffCanvas ? 12 : 3}
                >
```

Adds a litte top margin to the search field in the sidebar. Also covers the case that you enlarge the window while the sidebar is open.